lib/pages/transactions_page/transactions_page.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+
+import 'widgets/custom_sliver_delegate.dart';
+import 'widgets/categories_tab.dart';
+import 'widgets/accounts_tab.dart';
+import 'widgets/list_tab.dart';
+
+class TransactionsPage extends StatefulWidget {
+  @override
+  _TransactionsPageState createState() => _TransactionsPageState();
+}
+
+class _TransactionsPageState extends State<TransactionsPage>
+    with SingleTickerProviderStateMixin {
+  static const List<Tab> myTabs = <Tab>[
+    Tab(text: "Elenco", height: 35),
```

Comment:
Can you use the english language?

Reply:
Exactly, maybe a good choice would be to write in English.
What do you think about it?

Reply:
IMO we should build the app in English as the default language to make it more international. Later on, we can add support for different languages including Italian. However, we should probably discuss this on Discord as a project-wide change.

Reply:
Yes, probably we should discuss about it on Discord. But IMHO it is the best choice. :D

Reply:
I changed all titles to English